blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 3mdeb open-source contributions
+abstract: 'We are excited to share with you the open-source contributions
+           of our team. We have been actively contributing to a number of
+           open-source projects that have a significant impact on the industry.
+           Through these contributions, we have been able to help improve the
+           overall functionality and security of these projects, while also
+           gaining valuable experience and knowledge that we can apply to our
+           client work.'
+cover: /covers/image-file.png
+author: norbert.kaminski
+layout: post
+published: true
+date: 2023-04-27
+archives: "2023"
+
+tags:
+  - contribution
+  - coreboot
+  - fwupd
+  - trenchboot
+categories:
+  - Manufacturing
+
+---
+
+We are excited to share with you the open-source contributions of our team.
+We have been actively contributing to a number of open-source projects that have
+a significant impact on the industry. Through these contributions, we have been
+able to help improve the overall functionality and security of these projects,
+while also gaining valuable experience and knowledge that we can apply to our
+client work.
```

Comment:
Abstract repeated, why?

Please reflect on the 3mdeb situation during the contribution period you want to discuss.

Reply:
https://github.com/3mdeb/news-and-ideas/pull/519/commits/68a72cdebcee326365faf61a3b7861f718765122#diff-5dff579917e1df023c3d68f093f637f7fb497ee2f22650cc786ac30a195a7eceR30